### PR TITLE
Bug filter with or works incorrect

### LINKED ISSUE #365: Filter Q works incorrect.
Example: filter(Q() | Q()).filter(Q()). It's returns filter(Q()| Q())

### PR DESCRIPTION
**Pull request by MardanovTimur**
closes #365

### CODE REVIEW

**Comment by MardanovTimur** [comment]
Hi! It is necessary to write new tests.

**Comment by robinedwards** [comment]
Thanks just added one minor comment

File: neomodel/match.py [hunk 1]
```diff
@@ -340,7 +340,10 @@ def _parse_q_filters(self, ident, q):
         target = []
         for child in q.children:
             if isinstance(child, QBase):
-                target.append(self._parse_q_filters(ident, child))
+                q_childs = self._parse_q_filters(ident, child)
```
Comment (MardanovTimur) [inline neomodel/match.py]: not only minor comment. This block of code - for place the "(" and ")" in correct places. In both sides of OR's conditions

**Comment by MardanovTimur** [comment]
This pull request is very important for neomodels Q's. 
Because 
```
     .filter(arg1=val1).filter(Q(arg2=val2)|Q(arg3=val3))
```
 works  incorrect

**Comment by MardanovTimur** [comment]
It should be included in the next release

**Comment by robinedwards** [comment]
Yes can you answer why the count() is changed from == to >= (see comment above)

**Comment by MardanovTimur** [comment]
Because if node linked with another node more than 1 rels count>=1

**Comment by robinedwards** [comment]
Super thank you!!

**Pull request merged**